### ISSUE TITLE
ofxhome (and therefor ofxclient) doesn't seem to work anymore

### ISSUE THREAD

**Issue by beorn** [egh/ledger-autosync#141]
See https://github.com/captin411/ofxclient/issues/39

Perhaps use ofxtools instead? https://github.com/csingley/ofxtools

**Comment by egh** [comment]
You should still be able to use downloaded ofx files with ledger-autosync. Banks seem to have been moving away from support ofx servers for a while, sadly.

ofxtools does look nice, and if I was starting ledger-autosync today I'd probably use it instead of ofxclient. But it also looks like it uses ofxhome - so it probably has the same problem ofxclient has.

Additionally, I don't really have the time to work on this at the moment. If somebody wants to switch ledger-autosync to use ofxtools or to improve the ofx downloading, I'm more than open to merging PRs.

**Comment by beorn** [comment]
I think ofxtools includes the database from ofxhome: https://ofxtools.readthedocs.io/en/latest/client.html#basic-connectivity-requesting-an-ofx-profile

And the DNS entry for www.ofxhome.com doesn't even resolve anymore - unless I'm in some kind of DNS black hole these days :)

**Comment by egh** [comment]
Yes, ofxhome seems completely down. But you should still be able to import OFX files using ledger-autosync
